recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
```

Comment:
Yep, knew I had seen it before.

Was added by @yungyuc in this [commit]( https://github.com/conda/conda-recipes/commit/39a721784eee55c04966abdf8c082c430c5c98d5 ) who kindly submitted it upstream. Here is the link to the upstream submission ( http://www.graphviz.org/mantisbt/view.php?id=2448 ). Please just include that in a comment above the patch.